docs/docs/references/applications/README.mdx

```diff
@@ -0,0 +1,67 @@
+import ApplicationTypesIntro from '../../fragments/_application_types_intro.mdx';
+
+# 📱 Applications
+
+## Introduction
+
+Each real-world application that requires user authentication using Logto needs to be registered as an application in Logto.
+
+## Application ID
+
+_Application ID_ is a unique auto-generated identifier to identify your application with the Logto application.
```

Comment:
nit: auto-generated identifier -> auto-generated key ? or sth else.  identifier to identify is a duplicate usage of identify terms.